packages/frontend/src/lib/table/model/ModelColumnActions.svelte

```diff
@@ -4,19 +4,12 @@ import { faTrash } from "@fortawesome/free-solid-svg-icons";
 import { faFolderOpen } from "@fortawesome/free-solid-svg-icons";
 import ListItemButtonIcon from "../../button/ListItemButtonIcon.svelte";
 import { studioClient } from "/@/utils/client";
-import Modal from "../../Modal.svelte";
-import Button from "../../button/Button.svelte";
 export let object: ModelInfo;
 
-let deleteConfirmVisible: boolean = false;
-
-function deleteModel() {
-  deleteConfirmVisible= true;
-}
-
-async function goDeleteModel() {
-  await studioClient.deleteLocalModel(object.id);
-  deleteConfirmVisible= false;
+const deleteModel = () => {
```

Comment:
What is the rationale for using this pattern and not function ?

Reply:
There are pros and cons, but in the end, just a matter of preferences

Reply:
I would say it's about consistency with existing code ?

I mean, if for example someone love tabs and another spaces, or one prefer 2 spaces rather than 4 spaces, etc if everyone use the matter of preferences then it could produce an overall inconsistent code.